api/src/database/identity-docs.ts

```diff
@@ -35,7 +35,8 @@ export const saveIdentity = async (identity: IdentityJsonUpdate, secret: string)
 
 	const document = {
 		_id: encryptedIdentity?.doc?.id,
-		...encryptedIdentity
+		...encryptedIdentity,
+		created: new Date()
```

Comment:
Is the date format defined somewhere? 
By default it will create something like `Thu Jun 10 2021 18:04:22 GMT+0200 (Central European Summer Time)
`

Reply:
date will be stored as Date type in the mongodb, when sending dates to the client as string they will be formatted using date-fns